openedx/core/lib/edx_api_utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 def get_edx_api_data(api_config, user, resource,
-                     api=None, resource_id=None, querystring=None, cache_key=None):
+                     api=None, resource_id=None, querystring=None, cache_key=None, collection=True):
```

Comment:
I'm sort of thinking of this along the lines of DRF where you specify `many=True` for the serializers -- see [here](http://www.django-rest-framework.org/api-guide/relations/)